README.md

```diff
@@ -88,8 +88,8 @@ We will know we are successful, if we can increase worker productivity and moral
 | There is a risk that we won't be able to deploy to production frequently enough that we can work in an agile fashion.  | 2  | Derisked  |
 | There is a risk that vendors won't be able to easily work with the legacy ARIES code and existing authentication methods. | 5  | Not started  |
 | There is a landscape of existing solutions that could potentially be leveraged instead of building from scratch.  |  2 |  Not started |
-| There is a risk that the ATO process will prevent us from being able to deploy continuously to a production environment.  | 2  | Derisked  |
-| There is a risk that by having multiple vendors contributing to the same codebase, we increase the complexity of integration. There could be problems that arise specifically from the logistics and timing of multiple contributors.   | 2  |  Derisked |
+| There is a risk that the ATO process will prevent us from being able to deploy continuously to a production environment.  | 5  | In progress  |
+| There is a risk that by having multiple vendors contributing to the same codebase, we increase the complexity of integration. There could be problems that arise specifically from the logistics and timing of multiple contributors.   | 3  |  Not started |
```

Comment:
I'm not sure if I'd say this is "not started". I agree that it should not be "Derisked". But some of the VSTS prep and several other technical and design choices are working towards this. But, I'm not firm on this point. I could see it either way.

Reply:
Yeah, I guess my initial thinking on this was that it wouldn't be derisked until there was a 2nd vendor contributing code through VSTS, but you're right that this work has been started. I think changing from `Not started` to `In progress`makes the most sense.

Reply:
@edmullen Fixed it.